packages/core/src/app/checkout/getCheckoutStepStatuses.ts

```diff
@@ -124,6 +125,15 @@ const getBillingStepStatus = createSelector(
             };
         }
 
+        const isGooglePayBillingAddressEditingEnabled = isExperimentEnabled(
+            config?.checkoutSettings,
+            'STRIPE-546.allow_billing_address_editing_for_all_Google_Pay_providers',
+        );
+        const isUsingGooglePay =
+            isGooglePayBillingAddressEditingEnabled && (checkout && checkout.payments
+                ? checkout.payments.some((payment) => (payment?.providerId || '').startsWith('googlepay'))
```

Comment:
Are you sure that `.startsWith('googlepay')` is enough and we don't have providers with `providerId` for GPay that starts from some other prefix?

Reply:
I've checked our Google pay provider names, and it looks like all this providers starts with `googlepay`.
<img width="969" alt="Screenshot 2024-12-11 at 16 11 45" src="https://github.com/user-attachments/assets/3d71ea22-1821-4711-a990-ca5d2783e06d" />